### PR TITLE
Added support for posting a body of content

### PR DESCRIPTION
**Pull request by drewhemm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +142,15 @@ scrape_configs:
         regex: .*
         target_label: __address__
         replacement: 127.0.0.1:9115  # Blackbox exporter.
+
+  - job_name: 'blackbox-post'
+    metrics_path: /probe
+    params:
+      body: ['{}'] # Content, type must match the Content-Type header configured in the module
```
Comment (brian-brazil) [inline README.md]: All parameters must come from the config.

Comment (drewhemm) [inline README.md]: May I ask why? It's more flexible having it in the scrape config. We have multiple endpoints that we scrape, at the moment with the same body, but if we needed different body content, we would need to have multiple module declarations and could no longer use generic names such as http_json_post_2xx, we'd need http_json_post_2xx_service_a and http_json_post_2xx_service_b etc. 

Kind regards

Andrew

Comment (brian-brazil) [inline README.md]: This is how we approach everything for the blackbox exporter. Each config is meant to be a test of a particular thing such as a service, which you then run against multiple targets.

Comment (drewhemm) [inline README.md]: Okay, I will adjust and be back shortly :)

**Comment by drewhemm** [comment]
I will squash my changes into a single commit, then open a new pull request.

**Pull request closed without merging**